### ISSUE TITLE
[^1.12] [Suggestion] Support for Markdown Syntax?

### ISSUE THREAD

**Issue by SnowShock35** [GenDeathrow/MPUtils#19]
Not sure if this would possible but would be kinda cool if you could pull it off. Just thought I'd leave an issue here just in case. 🙂

**Comment by GenDeathrow** [comment]
As cool as that is, minecraft handles the text generation in a way that doesn't work super well. The code I wrote for mputils would need a pretty hefty change to pull it off. I keep this here. but that would be a long way down the road.

**Comment by SnowShock35** [comment]
Yeah that's what I expected. But glad you have an open mind to consider it in the future :-)
